src/Tempest/Reflection/src/ClassReflector.php

```diff
@@ -72,6 +72,11 @@ public function getName(): string
         return $this->reflectionClass->getName();
     }
 
+    public function getFilePath(): string|false
```

Comment:
Let's change this to `string`. It'll only return `false` if we're reflecting over internal PHP classes. I'd rather throw an exception in that case:

```php
$path = $this->reflectionClass->getFileName();

if ($path === false)
{
    // throw
}

return $path;
```

Reply:
I'll check this out because I don't remember tweaking on this, so maybe a vestige of the `feat/publish`